### ISSUE TITLE
Alt + WASD will not turn to face your previous direction of movement

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#81524]
Reporting client version: 515.1630

## Round ID:
[224084](https://scrubby.melonmesa.com/round/224084)

## Testmerges:
- [bindable action buttons](https://www.github.com/tgstation/tgstation/pull/81371)

## Reproduction:

Spawn yourself in
Move downwards a couple of tiles with 's'
Hold ALT (is it control for some people?) and cycle through the WASD movement keys, you will notice you "turn to face" the direction you press other than ALT+S, you can not 'turn to face' south.
Move upwards a few tiles with 'w'
Again hold ALT and cycle through the movements, this time you will find you can not "turn to face" the up screen direction (alt+w)

Basically whatever direction you last moved you can no longer "turn to face"

**Comment by SyncIt21** [comment]
Duplicate #81501?

**Comment by LemonInTheDark** [comment]
yes
